reports/report_builder.py

```diff
@@ -3131,7 +3131,33 @@ def ws_104(self, ws):
         return
     
     def ws_105(self, ws):
-        return
+        """
+        Cols: Survivors
+        Rows: Major topic, covid stories only, Sex of source
+        """
+        r = 6
+        self.write_col_headings(ws, SURVIVOR_OF)
+
+        for major_topic, topic_ids in GROUP_TOPICS_MAP.items():
+            counts = Counter()
+            for media_type, model in person_models.items():
+                if 'survivor_of' in [field_name.name for field_name in model._meta.get_fields()]:
+                    rows = model.objects \
+                            .values('sex', 'survivor_of') \
+                            .filter(**{model.sheet_name() + '__covid19': 1}) \
+                            .filter(**{model.sheet_name() + '__country__in': self.country_list}) \
+                            .exclude(survivor_of=None) \
+                            .filter(**{model.sheet_name() + '__topic__in': topic_ids}) \
+                            .filter(sex__in=self.male_female_ids) \
```

Comment:
Shouldn't we be using all genders?

Reply:
That's what I thought @esirK  but try searching for sheets that show gender, most just show Male / Female or just Female.